src/sfdxProject.ts

```diff
@@ -72,7 +72,7 @@ export type ProjectJson = ConfigContents & {
  * be in a top level property that represents your project or plugin.
  *
  * ```
- * const project = await SfdxProjectJson.retrieve();
+ * const project = await retrieveSfdxProjectJson();
```

Comment:
@RodEsp Since `retrieveSfdxProjectJson` is an instance method of `SfdxProject`, this example should include how to construct the SfdxProject instance. Also, the docs for `retrieveSfdxProjectJson` state one should use `resolveProjectConfig` instead.

@mdonnalley I grabbed this snippet from `SfdxProject` API docs. Will this suffice?

```
const project = await SfdxProject.resolve();
const projectJson = await project.resolveProjectConfig();
```

Reply:
You're totally right. I accidentally missed those belonged to two separate classes.